lib/internal/bootstrap_node.js

```diff
@@ -225,12 +225,27 @@
     global.setTimeout = timers.setTimeout;
   }
 
+  function setupConsoleAPI(addCommandLineAPIMethod) {
+    addCommandLineAPIMethod('require', function require(request) {
```

Comment:
Instead of passing, this could just access `process.inspector.addCommandLineAPIMethod` directly, whcih would be preferred.